README.md

```diff
@@ -20,6 +20,15 @@ Níže najdete interní dokumenty Appliftingu, dle kterých se u nás řídíme.
 4. 📜🤝📜 [**Příručka pro řešení konfliktů**](./cs/managing-conflicts.md) - V tomto dokumentu je popis procesu, který se v Appliftingu používá pro řešení pracovních i osobních konfliktů
 5. 📜🏛📜 [**Jak funguje Rada starších**](https://www.notion.so/applifting/How-Council-of-Elders-Works-8dff507d7f9247fdb68c0d4029331628) - Dokument shrnující pravomoci našeho boardu a způsob, jakým do něj Applifteři volí členy.
 
+## Pomáháme ostatním
+
+Jdeme za hranici naší práce. Chceme mít pozitivní vliv na komunity kolem nás a systematicky se tomu věnujeme.
+
+* 🤗 [**Spirit of Applifting**](https://www.notion.so/applifting/Spirit-of-Applifting-1cbf1c9247cc442da73f768ffcedadc4) - Applifteři mohou pomáhat ostatním za peníze Appliftingu.
+* 💰 **Malou část zisku darujeme charitám** - Každý rok dáváme 0,5% zisku na charity. Obdarované organizace jsou vybírány Appliftery.
```

Comment:
Obdarované organizace jsou vybírány Appliftery v rámci Vánočních odměn. Každý tak může přispět na to, co mu dává smysl.

Reply:
Nikdo mimo nás nezná "Vánoční odměny" takže bych je do toho asi netahal.

Reply:
Ouk, v tom případě bych tu větu rozvinul nějak jinak, protože takhle je děsně strohá :)